pkg/addons/data/Deployment.apps/openshift-infra/customer-admin-reconciler.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: customer-admin-reconciler
+  namespace: openshift-infra
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: customer-admin-reconciler
+  template:
+    metadata:
+      labels:
+        app: customer-admin-reconciler
+    spec:
+      serviceAccountName: customer-admin-reconciler
+      containers:
+      - args:
+        - |
+          set -o errexit
+          set -o nounset
+          set -o pipefail
+
+          cat <<'SCRIPT' >/tmp/reconcile
+          #!/bin/bash -x
+          set -o errexit
+          set -o nounset
+          set -o pipefail
+
+          project=$1
+          echo "Project: ${project}"
+
+          # do nothing for whitelisted projects
+          for ns in '^openshift-.*$' '^openshift$' '^kube-.*$' '^default$';
+          do
+            if [[ "${project}" =~ $ns ]]; then
+              exit 0
+            fi
+          done
+
+          oc auth reconcile -f /tmp/rolebindings -n "${project}"
+
+          ##### END HERE #######
+          SCRIPT
+          chmod u+x /tmp/reconcile
+
+          exec oc observe project --maximum-errors=1 --resync-period=15m -- /tmp/reconcile
```

Comment:
This should be namespace, not project.

BTW this will fail to notice role binding changes until it resyncs. That is probably fine.

Reply:
@enj, is namespace better than project?

Reply:
Agreed that the rolebindings changes are not picked up until next sync.  I think that is acceptable.

Reply:
Will update.